app/views/service_providers/all.html.erb

```diff
@@ -12,22 +12,32 @@
       <th scope="col">Active</th>
       <th scope="col">Cert Exp</th>
       <th scope="col">Created</th>
-      <th scope="col">In Production</th>
     </tr>
   </thead>
   <tbody>
-    <% @service_providers.each do |app| %>
+    <% @service_providers.select { |sp| sp.prod_config == true }.each do |app| %>
+      <%= render partial: 'service_provider_row', locals: { app: app, show_created_at: true } %>
+    <% end %>
+  </tbody>
+</table>
+
+
+<h1 class="usa-display">Non-Production Apps</h1>
```

Comment:
Open to comments on the name of this header.

Reply:
How about 'Sandbox Apps' ?

Reply:
I like that more! We have a new leader.